<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="7" column="5" rowspan="5" colspan="2">
    <widget class="QLCDNumber" name="lcd_speed">
     <property name="font">
      <font>
       <pointsize>23</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="4">
    <spacer name="horizontalSpacer_7">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>120</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>120</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="1" rowspan="5" colspan="3">
    <widget class="QLCDNumber" name="lcd_rot">
     <property name="font">
      <font>
       <pointsize>23</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="7" column="7">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>120</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="10">
    <widget class="QComboBox" name="comboBoxDevices"/>
   </item>
   <item row="8" column="10">
    <widget class="QPushButton" name="pushButtonDisconnect">
     <property name="text">
      <string>Rozłącz</string>
     </property>
    </widget>
   </item>
   <item row="5" column="8">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>420</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="5">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="10">
    <widget class="QPushButton" name="pushButtonConnect">
     <property name="text">
      <string>Połącz</string>
     </property>
    </widget>
   </item>
   <item row="7" column="8" rowspan="5">
    <widget class="QLCDNumber" name="lcd_dist">
     <property name="font">
      <font>
       <pointsize>23</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="6" column="8">
    <widget class="QProgressBar" name="progressBar">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>24</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="10">
    <widget class="QPushButton" name="pushButtonSearch">
     <property name="text">
      <string>Szukaj</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="7">
    <spacer name="horizontalSpacer_8">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
